quickstarts/azure/azure-stream-analytic/config.yml

```diff
@@ -0,0 +1,39 @@
+slug: azure-stream-analytic
```

Comment:
Should the slug actually be "azure-stream-analytics" to match the name of the service?